src/styles/mocto-theme.scss

```diff
@@ -21,9 +21,12 @@ $mocto-disabled-light: #E9E9E9;
 $mocto-disabled: #C4C4C4;
 $mocto-disabled-dark: #7B7B7B;
 
+$mocto-box-height: 19px;
```

Comment:
This is also in another PR.

Reply:
I made a comment in your other PR about this - it's also related to the comment above about using spacing tokens

Reply:
I will adjust this in the other PR.